inductiva/api/methods.py

```diff
@@ -29,9 +29,12 @@
 from inductiva.utils import format_utils, files
 
 
-def validate_api_key(api_key: Optional[str]) -> Configuration:
+def validate_api_key() -> Configuration:
```

Comment:
let's rename this method. It is not validating the key - it is building an  api_config object.
Validation is only a subproduct of the process.